doc/examples/segmentation/plot_metrics.py

```diff
@@ -0,0 +1,145 @@
+# :func:`skimage.segmentation.morphological_geodesic_active_contour`, a method
+# that generally produces good results, but requires a long time to converge on
+# a good answer. We purposefully cut short the procedure at 100 iterations, so
+# that the final result is *undersegmented*, meaning that many regions are
+# merged into one segment. We will see the corresponding effect on the
+# segmentation metrics.
+
+image = img_as_float(image)
+gradient = inverse_gaussian_gradient(image)
+init_ls = np.zeros(image.shape, dtype=np.int8)
+init_ls[10:-10, 10:-10] = 1
+im_test3 = morphological_geodesic_active_contour(gradient, iterations=100,
+                                                 init_level_set=init_ls,
+                                                 smoothing=1, balloon=-1,
+                                                 threshold=0.69)
+im_test3 = label(im_test3)
+
+method_names = ['Compact watershed', 'Canny filter',
+                'Morphological Geodesic Active Contours']
+short_method_names = ['Compact WS', 'Canny', 'GAC']
+
+precision_list = []
+recall_list = []
+split_list = []
+merge_list = []
+for name, im_test in zip(method_names, [im_test1, im_test2, im_test3]):
+    error, precision, recall = adapted_rand_error(im_true, im_test)
+    splits, merges = variation_of_information(im_true, im_test)
+    split_list.append(splits)
+    merge_list.append(merges)
+    precision_list.append(precision)
+    recall_list.append(recall)
+    print('\n## Method: {}'.format(name))
+    print('Adapted Rand error: {}'.format(error))
+    print('Adapted Rand precision: {}'.format(precision))
+    print('Adapted Rand recall: {}'.format(recall))
+    print('False Splits: {}'.format(splits))
+    print('False Merges: {}'.format(merges))
+
+fig, axes = plt.subplots(2, 3, figsize=(4, 6), constrained_layout=True)
+ax = axes.ravel()
+
+ax[0].scatter(merge_list, split_list)
+for i, txt in enumerate(short_method_names):
+    ax[0].annotate(txt, (merge_list[i], split_list[i]),
+                   verticalalignment='center')
+ax[0].set_xlabel('False Merges (bits)')
+ax[0].set_ylabel('False Splits (bits)')
+ax[0].set_title('Split Variation of Information')
+
+ax[1].scatter(precision_list, recall_list)
+for i, txt in enumerate(short_method_names):
+    ax[1].annotate(txt, (precision_list[i], recall_list[i]),
+                   verticalalignment='center')
+ax[1].set_xlabel('Precision')
+ax[1].set_ylabel('Recall')
+ax[1].set_title('Adapted Rand precision vs. recall')
+ax[1].set_xlim(0, 1)
+ax[1].set_ylim(0, 1)
+
+ax[2].imshow(mark_boundaries(image, im_true))
+ax[2].set_title('True Segmentation')
+ax[2].set_axis_off()
+
+ax[3].imshow(mark_boundaries(image, im_test1))
+ax[3].set_title('Compact Watershed')
+ax[3].set_axis_off()
+
+ax[4].imshow(mark_boundaries(image, im_test2))
+ax[4].set_title('Edge Detection')
+ax[4].set_axis_off()
+
+ax[5].imshow(mark_boundaries(image, im_test3))
+ax[5].set_title('Morphological GAC')
+ax[5].set_axis_off()
+
+plt.show()
```

Comment:
This is how the plot looks like for me.

![sphx_glr_plot_metrics_001](https://user-images.githubusercontent.com/335370/61575850-ab47b180-aad1-11e9-878f-eb3b2a37d9e8.png)

Reply:
Ah yikes, I changed the figure size and it looks good on my end.